client/src/components/manageProjects/editProject.js

```diff
@@ -207,4 +198,21 @@ const EditProject = ({
   );
 };
 
+function RecurringEvent({event, setSelectedEvent}) {
+
+  return (
+    <li key={`${event.event_id}`}>
+      <button type="button" onClick={() => {console.log(event);setSelectedEvent(event)}}>
+        <div>{event.name}</div>
+        <div className="event-list-details">
+          {`${event.dayOfTheWeek}, ${event.startTime} - ${event.endTime}; ${event.eventType}`}
+          <div className="edit-icon"><EditIcon /></div>
+        </div>
+        <div className="event-list-description">{`Can uses check into this event now?: ${event.checkInReady ? "Yes" : "No"}`}</div>
```

Comment:
`setSelectedEvent` designates which event should be editable in the modal rendered by `<EditMeetingTimes />`